corehq/apps/app_manager/tests/test_bulk_ui_translation.py

```diff
@@ -35,7 +36,12 @@ def test_no_change(self):
         headers = (('translations', ('property', 'en')),)
         f = self._build_translation_download_file(headers)
```

Comment:
You could make this function return a real file and avoid all other changes to this file.